### ISSUE TITLE
"My activity" and "Logout" drop-down menu is not visible on Mobile

### ISSUE THREAD

**Issue by softvision-oana-arbuzov** [webcompat/webcompat.com#2163]
URL: https://staging.webcompat.com/
Browser / Version: Chrome Desktop 64.0.3282.167, Chrome Mobile 64.0.3282.137
Operating System: Google Pixel (Android 8.0.0) - 1080 x 1920 pixels (~441 ppi pixel density)

**Prerequisites:**
1. Github account available and signed in.

**Steps to Reproduce:**
1. Navigate to https://staging.webcompat.com/
2. Tap the arrow on the right side of the Github user.
3. Observe behavior.

**Expected Behavior:**
Menu with "My activity" and "Logout" option is displayed.

**Actual Behavior:**
Menu with "My activity" and "Logout" option is not visible.

**Note**
1. Not reproducible on Desktop.
2. Affected area:
```html
<button class="nav-button nav-link" aria-pressed="false">
	<img class="login-avatar-img" src="https://avatars0.githubusercontent.com/u/12184325?v=40" alt="User Avatar">
		<svg class="icon nav-icon" viewBox="0 0 30 30" role="presentation">
			<use href="#svg-chevron-right">dropdown</use>
		</svg>
</button>
```

Watchers:
@softvision-sergiulogigan
@softvision-oana-arbuzov

![screenshot-1519051946376](https://user-images.githubusercontent.com/12184325/36383883-2a8f0600-1596-11e8-9713-eaed9a736095.jpg)

**Comment by softvision-oana-arbuzov** [comment]
This issue is related to #2079.

**Comment by zoepage** [comment]
This needs a different positioning for the other break points as they are missing right now.

**Comment by magsout** [comment]
@zoepage on my localhost this bug is fixed, or I missed something ?

**Comment by zoepage** [comment]
Did you check on chrome @magsout ? I could reproduce today 🤔

**Comment by zoepage** [comment]
Dang, I missread on my phone. Sorry! THis should be fixed by #2172

**Comment by magsout** [comment]
👍
